service/src/test/java/uk/gov/hmcts/reform/fpl/service/cmo/ReviewCMOServiceTest.java

```diff
@@ -261,7 +261,7 @@ void shouldThrowExceptionIfSealedCMOListIsEmpty() {
     }
 
     @Test
-    void shouldReturnIssueResolutionStateWhenNextHearingTypeIsIssueResolutionAndFeatureToggleIsToggledOn() {
+    void shouldReturnCaseManagementStateWhenNextHearingTypeIsIssueResolutionAndFeatureToggleIsToggledOn() {
```

Comment:
there is method shouldReturnCurrentStateWhenNextHearingIsNotOfTypeIssueResolutionOrFinal which should be renamed

Reply:
Few of them actually, i'll change.